examples/local/202_move_tables.sh

```diff
@@ -19,4 +19,7 @@
 
 source ../common/env.sh
 
-vtctlclient MoveTables -- --source commerce --tables 'customer,corder' Create customer.commerce2customer
+vtctldclient movetables --workflow "commerce2customer" --target-keyspace "customer" create --source-keyspace "commerce" --tables "customer,corder"
```

Comment:
Let's stick to CamelCase (MoveTables) in the new commands: makes it consistent with current usage and documentation. I see we already have these command aliases setup.